### DISCUSSION TITLE
How to configure BlackHole to send Stream Deck soundboard to Zoom Meetings?

### DISCUSSION THREAD

**Discussion by squiter** [ExistentialAudio/BlackHole#722] [Q&A] [answered]
Hey folks, sorry if this was already answered, but I couldn't find it here.

I want to use the Stream Deck's soundboard during some of my Zoom meetings, but I couldn't figure out how to do it, I'll explain here what I tryed:

- I created a Multi Ouput device with my headphone first and then the BlackHole;
- I created an Aggregate Device with my Yeti's microfone first and then the BlackHole;

Then, on Zoom, I tried to select the Aggregate Device as my input source, but it doesn't work... what am I missing?

I'll paste here my setup:
![image](https://user-images.githubusercontent.com/238017/234423572-8ff7581b-51e7-4360-b187-2393dcfb5785.png)
![image](https://user-images.githubusercontent.com/238017/234423743-196120fa-f6c9-42b5-944c-8963627514b6.png)

On the screenshots, the default input device isn't set to the Aggregate Device, but I tested selecting it already

**Comment by devinroth** [comment]
Try switching BlackHole and the Yeti.

That being said, an aggregate is not a multi-input device. You need to merge them together.

Devin

**Reply by squiter** [reply]
Hey man, thanks for you fast reply! :)

What do you mean by "merge them together"?

**Reply by squiter** [reply]
I changed the order of the BlackHole and Yeti, and the soundboard goes to the "mic" of my meetings, but I lost my Yeti :(

**Comment by devinroth** [comment, marked as answer]
An aggregate doesn’t mix the audio devices together. So your aggregate becomes a 3 channel audio with the Yeti on channel 3. That’s why you don’t hear it.

If you’re on Ventura try this app we’re developing called Dipper. It will solve your problem.

https://discord.com/channels/775472045865173023/1047774289480400986

Devin

**Reply by squiter** [reply]
Awesome, it works very well! Thank you man
